src/base/utils/net.cpp

```diff
@@ -0,0 +1,13 @@
+#include "net.h"
+#include <QHostAddress>
+
+namespace Utils
+{
+    namespace Net
+    {
+        bool isValidIP(const QString &ip)
+        {
+            return !QHostAddress(ip).isNull();
```

Comment:
So, now which is it? I mean now that you simplified it, do you still think it will be useful as wrapping one thing only?